src/ProtocolZoo/ProtocolZoo.jl

```diff
@@ -212,16 +212,19 @@ end
     end
 end
 
-function findswapablequbits(net,node) # TODO parameterize the query predicates and the findmin/findmax
+function findswapablequbits(net, node, pred_low, pred_high) # TODO parameterize the query predicates and the findmin/findmax
     reg = net[node]
+    
+    l = typeof(pred_low) == Wildcard ? <(node) : pred_low
+    h = typeof(pred_high) == Wildcard ? >(node) : pred_high
```

Comment:
Do we need the above two lines?

Reply:
I do not think we should have these. My guess is that you are trying to set some defaults here? But I do not think we should treat the behavior for a simple chain as the automatic default.

Related to this, make sure that the examples what use this code are updated to explicitly use the `>(node)` and `<(node)` predicates.